shared/middleware/files_api.rb

```diff
@@ -338,7 +338,10 @@ def put_file(endpoint, encrypted_channel_id, filename, body)
     end
 
     # Block libraries with PII/profanity from being published.
-    return bad_request if endpoint == 'libraries' && ShareFiltering.find_failure(body, request.locale)
+    share_failure = ShareFiltering.find_failure(body, request.locale)
+    # TODO(JillianK): we are temporarily ignoring address share failures because our address detection is very broken.
```

Comment:
nit: add a link to the jira ticket (https://codedotorg.atlassian.net/browse/STAR-1361) to give more context about what's broken here